This file is part of mingw-cross-env.
See doc/index.html for further information.

This patch has been homemade!

--- armadillo-2.2.3.orig/CMakeLists.txt	2011-08-25 06:24:19.000000000 +0300
+++ armadillo-2.2.3/CMakeLists.txt	2011-10-15 15:15:42.000000000 +0300
@@ -251,7 +251,7 @@
 #set(CMAKE_INSTALL_RPATH_USE_LINK_PATH  TRUE)
 
 
-add_library( armadillo SHARED src/pull_libs )
+add_library( armadillo STATIC src/pull_libs )
 target_link_libraries( armadillo ${ARMA_LIBS} )
 
 set_target_properties(armadillo PROPERTIES VERSION ${ARMA_MAJOR}.${ARMA_MINOR}.${ARMA_PATCH} SOVERSION 2)
